public/app/features/browse-dashboards/components/FolderActionsButton.tsx

```diff
@@ -141,6 +135,18 @@ export function FolderActionsButton({ folder }: Props) {
           <Permissions resource="folders" resourceId={folder.uid} canSetPermissions={canSetPermissions} />
         </Drawer>
       )}
+      {showDeleteProvisionedFolderDrawer && (
+        <Drawer
+          title={t('browse-dashboards.action.delete-provisioned-folder', 'Delete Provisioned Folder')}
```

Comment:
```suggestion
          title={t('browse-dashboards.action.delete-provisioned-folder', 'Delete provisioned folder')}
```
Forgot to mention in the previous PR, but the recommendation from the docs is not to use title case.